CONTRIBUTING.md

```diff
@@ -3,24 +3,25 @@
 ## Create the docs locally
 
 ``` python
-conda env update -f environment.yml
+conda env create --file environment.yml
 conda activate geocat-applications
 make clean html
 open _build/html/index.html
 ```
+If the `_build` folder is generated, but `open` command cannot open the html, you can navigate to the `index.html` document and right-click to `Open With` your desired browser.
```

Comment:
I wonder if this is a mac thing because `open _build/html/index.html` works as expected for me

Reply:
It might be, it gave me issues running on Linux, even though the html was being generated

Reply:
Hmm, yeah. How do you feel about leaving it like this here and making a separate issue to expand this section into a proper "Development workflow" section (like in [geocat-comp](https://geocat-comp.readthedocs.io/en/stable/contrib.html#development-workflow-overview) or [xarray](https://docs.xarray.dev/en/stable/contributing.html#development-workflow))?

Reply:
Sounds good, will do